pkgs/applications/editors/vscode/extensions/vscode-utils.nix

```diff
@@ -88,7 +88,7 @@ let
         "vsix"
       ])
       // {
-        name = "${mktplcRef.publisher}-${mktplcRef.name}-${mktplcRef.version}";
+        name = "${mktplcRef.publisher}-${mktplcRef.name}";
```

Comment:
```suggestion
        pname = "${mktplcRef.publisher}-${mktplcRef.name}";
```

Shouldn't this be ^^ ?

Reply:
Yes, but that would be a breaking change for the `buildVscodeExtension` function.

Reply:
Fair enough.

Reply:
We could move this to after branch-of, add a changelog entry.

Reply:
Sure why not!